winui/Rating/Placeholder-value.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: Placeholder value in WinUI Rating control | Syncfusion
+description: Learn all about the placeholder feature in the Rating control to show the average rating value before the user provide their own rating.
+platform: WinUI
+control: SfRating
+documentation: ug
```

Comment:
Documentation: UG

Reply:
We have follow the small letter for all other controls.